src/lang/KclSingleton.ts

```diff
@@ -232,6 +233,12 @@ export class KclManager {
   async executeAst(args: ExecuteArgs = {}): Promise<void> {
     if (this.isExecuting) {
       this.executeIsStale = args
+      // Previous execution will be rejected and finish, execution will be marked as stale
+      // new executeAst will start. This will reject previous AST calls while the user is typing
+      // in the editor
+      this.engineCommandManager.rejectAllCommands(
+        `Force interrupt, executionIsStale, new AST requested`
```

Comment:
Can this be a constant so that it can be referenced when we detect whether execution was interrupted?